tests/src/test/scala/whisk/core/controller/test/ActivationsApiTests.scala

```diff
@@ -462,6 +462,58 @@ class ActivationsApiTests extends ControllerTestCommon with WhiskActivationsApi
     }
   }
 
+  it should "skip activations and return correct ones" in {
+    implicit val tid = transid()
+    val activations = (1 to 3).map { _ =>
+      WhiskActivation(
+        namespace,
+        aname(),
+        creds.subject,
+        ActivationId.generate(),
+        start = Instant.now,
+        end = Instant.now)
+    }.toList
+
+    try {
+      activations.foreach(storeActivation)
+      waitOnListActivationsInNamespace(namespace, 3)
+
+      Get(s"$collectionPath?skip=1") ~> Route.seal(routes(creds)) ~> check {
+        status should be(OK)
+        val activations = responseAs[List[JsObject]]
+        activations.length should be(2)
```

Comment:
Can we narrow this check by checking for the correct activations? Like:

```scala
responseActivations shouldBe activations.map(_.toJson).drop(1)
```

Reply:
There are two problems with this solution:
1) the two json results are not completely equal (one of them has more fields)
2) it looks like the activations might be returned in different order so it might be different every time
Comparing just the size of the results is more robust

Reply:
Hmm, that sounds fishy. If we define the timestamps for the activations that we save, the order should be guaranteed.

I buy the schema difference though, should we then just check for the activation id for example?

Reply:
Well, I'm just reporting what I'm seeing:)
Running this code twice:
`
Get(s"$collectionPath?skip=1") ~> Route.seal(routes(creds)) ~> check {
        status should be(OK)
        val activations = responseAs[List[JsObject]].map(_.fields("name"))
        println(s"Resp: $activations")
        }
`
Gives me:
`
1) Resp: List("activations_tests_name15", "activations_tests_name13")
2) Resp: List("activations_tests_name14", "activations_tests_name15")
`

Reply:
You might need to specifically set the start/end respectively to have each activation after the other. Granularity is milliseconds so it's plausible they all get the same timestamp in this test.